hermes/src/api/sse.rs

```diff
@@ -99,16 +108,30 @@ pub async fn price_stream_sse_handler(
         async move {
             match message {
                 Ok(event) => {
+                    // Check if the event is out-of-order and if unordered updates are not allowed
+                    if let AggregationEvent::OutOfOrder { .. } = event {
+                        if !params.allow_unordered {
+                            return Ok(Event::default().comment("Out-of-order event skipped"));
+                        }
+                    }
                     match handle_aggregation_event(
                         event,
                         state_clone,
                         price_ids_clone,
                         params.encoding,
                         params.parsed,
+                        params.benchmarks_only,
                     )
                     .await
                     {
-                        Ok(price_update) => Ok(Event::default().json_data(price_update).unwrap()),
+                        Ok(price_update) => {
+                            // Check if there is any data to send
+                            if price_update.binary.data.is_empty() {
+                                // No data to send, skip creating an event
+                                return Ok(Event::default().comment("No data to send"));
+                            }
+                            Ok(Event::default().json_data(price_update).unwrap())
```

Comment:
I'm a bit worried about having `unwraps`. can we add `unwrap_or_else(|e| error_event(e))`?